Task_4.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+while getopts "p:t:" flag; do
+    case "${flag}" in
+        p) path="${OPTARG}" ;;
+        t) text="${OPTARG}" ;;
+    esac
+done
+
+if [ $# -eq 0 ]; then
```

Comment:
```suggestion
if [ $# -neq 2 ]; then
```
You should check if two parameters were provided. Alternatively, create a function for script usage and check them later.

Reply:
In addition, try to avoid nested if else structures for better readability.

Reply:
Also change script name to 'task4.sh'.